lib/routes/components.js

```diff
@@ -10,10 +10,35 @@ module.exports = app => {
 	});
 
 	// Component listing page
-	app.get('/components', neverCache, (request, response) => {
-		response.render('components', {
-			title: `Components - ${app.origami.options.name}`
-		});
+	app.get('/components', neverCache, async (request, response, next) => {
+		try {
```

Comment:
Out of pure curiosity, does this type of logic not live in another file when it comes to origami apps?